app/containers/ViewCompany/index.js

```diff
@@ -0,0 +1,76 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
+import { compose } from 'redux';
+import { createStructuredSelector } from 'reselect';
+
+import injectReducer from 'utils/injectReducer';
+import injectSaga from 'utils/injectSaga';
+import Company from 'components/Company';
+import { selectCompany, selectErrorMessage } from './selectors';
+import { getCompany } from './actions';
+import reducer from './reducer';
+import saga from './saga';
+
+export class ViewCompany extends React.Component {
+  constructor(props) {
```

Comment:
You don't need this function. It already happens because you extend `React.Component`.